spec/lib/appsignal/hooks/redis_spec.rb

```diff
@@ -36,13 +36,21 @@
               # Stub Redis::Client class so that it doesn't perform an actual
               # Redis query. This class will be included (prepended) with the
               # AppSignal Redis integration.
-              stub_const("Redis::Client", Class.new do
+              stub_const("Redis::Client", Class.new(Redis::Client) do
                 def id
                   :stub_id
                 end
 
-                def process(_commands)
-                  :stub_process
+                def write(_command)
+                  nil
+                end
+
+                def read
+                  "value"
+                end
+
+                def call(*args)
+                  super
```

Comment:
need a different way of handling this. I guess I could call directly to write in the test and stub only that method, just like it was done with `process`

Reply:
That's okay for now I think. It's not perfect, but we also don't want to run the actual query here.